controllers/userController.js

```diff
@@ -0,0 +1,108 @@
+const userService = require("../services/dbUserServices");
+
+class UserController {
+  async getAllUsers(req, res) {
+    try {
+      const users = await userService.getAllUsers();
+      if (!users) {
```

Comment:
If you're trying to getAll records and no one has been found - empty array in response

Reply:
+